docs/zh_cn/tutorials/analysis/dygraph/latency_predictor.md

```diff
@@ -1,35 +1,99 @@
 # LatencyPredictor使用教程
 
-LatencyPredictor主要功能是根据提供的op-latency映射表，预估神经网络网络在特定硬件设备上的实际耗时。它基于Paddle-Lite开发，适用于使用Paddle-Lite部署的模型。映射表以key-value的形式存储，key包含了神经网络模型经过Paddle-Lite图优化后的各种融合op信息，value则代表在特定硬件上的实际耗时。
+延时预估器（LatencyPredictor）用于预估模型在特定硬件设备上的推理延时。在无需部署模型到实际环境的情况下，可以快速预估出多种部署环境和设置下的推理延时。当前，
+* 支持所有可以使用 Paddle Lite 部署的模型；
+* 支持预估 ARM CPU 上的模型耗时。
 
-## 使用方法
+## 1. 准备环境
+### 1.1 版本要求
+```bash
+python>=3.7
+PaddleSlim>=2.3.0
+```
+### 1.2 安装 PaddleSlim
+* 通过 pip install 的方式进行安装:
+```bash
+pip install paddleslim -i https://pypi.tuna.tsinghua.edu.cn/simple
+```
 
-1. 下载或自行编译opt优化工具
-2. 构建LatencyPredictor
-3. 定义模型和预测
+* 或者从源码安装最新版 PaddleSlim:
 
-### 1. 下载或自行编译opt优化工具
-1.1 下载提供的opt工具，可根据运行环境下载适用的opt，目前提供Mac平台([M1芯片](https://paddle-slim-models.bj.bcebos.com/LatencyPredictor/opt_M1_mac)，[Intel芯片](https://paddle-slim-models.bj.bcebos.com/LatencyPredictor/opt_intel_mac))和[Ubuntu](https://paddle-slim-models.bj.bcebos.com/LatencyPredictor/opt_ubuntu)平台的opt工具下载。
-1.2 也可以自行通过Paddle-Lite源码编译opt工具，具体请参考请参考Paddle-Lite[文档](https://paddle-lite.readthedocs.io/zh/latest/user_guides/model_optimize_tool.html)。编译时需要关闭Paddle-Lite的内存复用功能，即注释掉这[几行代码](https://github.com/PaddlePaddle/Paddle-Lite/blob/d76f45be989d3e01cebf2ac18e047cfd37d52666/lite/core/optimizer/optimizer.cc#L266-L268)。
+```bash
+git clone https://github.com/PaddlePaddle/PaddleSlim.git
+cd Paddleslim
+python3.7 setup.py install
+python3.7 -m pip install -r requirements.txt # 从requirements.txt安装依赖库
+```
 
-### 2. 构建LatencyPredictor
+## 2. 快速开始
+### 2.1 准备推理模型
+延时预估器通过读取推理模型文件（\*.pdmodel, \*.pdiparams）进行预估。以 MobileNetv1 为例，请从[这里](https://bj.bcebos.com/v1/paddlemodels/PaddleSlim/analysis/mobilenetv1.tar)下载推理模型文件。
+```bash
+wget https://bj.bcebos.com/v1/paddlemodels/PaddleSlim/analysis/mobilenetv1.tar
+tar -xf mobilenetv1.tar
+```
+使用自定义模型结构时，可参考[api文档](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/jit/save_cn.html#save)保存推理模型。
 
-提供opt工具路径，以及芯片和测试参数信息，LatencyPredictor会根据这些参数自动下载对应的映射表。如下所示，芯片为845芯片，测试线程数threads为4，测速模式power_mode为3，测试batchsize为1.
+### 2.2 预估推理延时
+构造 TableLatencyPredictor 类实例，并调用 predict 函数预估推理模型的延时。
 ```
 import paddleslim
 
-opt_path = {opt工具路径}
-predictor = paddleslim.TableLatencyPredictor(opt_path, hardware='845', threads=4, power_mode=3, batchsize=1)
+predictor = paddleslim.TableLatencyPredictor(table_file='SD710')
+latency = predictor.predict(model_file='mobilenetv1_fp32.pdmodel', param_file='mobilenetv1_fp32.pdiparams, data_type='fp32')
+print('predicted latency = {}ms'.format(latency))
 ```
+通过设置 table_file 来指定硬件信息，当前支持“SD625”、“SD710”、“SD845”三款骁龙芯片。
+> 注1：耗时是基于**保存推理模型时设定的输入形状**预估而得；
```

Comment:
What do you want to express in this sentence? What do you want the developers to take extra notice on?

It seems redundant if developers doesn't get any takeaways....